perf/stability/control-plane/test.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+setup() {
+  kubectl create ns twopods-istio
+  kubectl create ns invalid-namespace-0
+  kubectl label namespace invalid-namespace-0 istio-injection=enabled --overwrite
+  kubectl apply -f <(istioctl kube-inject -f ../../istio-install/tmp/istio-1.12.0/samples/sleep/sleep.yaml) -n invalid-namespace-0
```

Comment:
why use manual injection `kube-inject` when the namespace is already labeled with `istio-injection=enabled`?

Reply:
I added the `istio-injection=enabled` later and forgot to remove the `kube-inject`, I will fix this.

Reply:
is this fixed?

Reply:
haven't tested this specifically, will check that.

Reply:
this is still giving me some issues sometimes. I think kube-inject would be a better choice here because there might be a case where we are applying the injection on the namespace and before it gets injected, we try to deploy some workload.